app/models/katello/glue/provider.rb

```diff
@@ -83,7 +83,7 @@ def owner_upstream_export(upstream, zip_file_path, _options)
         end
 
         # Default to Red Hat
-        url = upstream['apiUrl'] || 'https://subscription.rhn.redhat.com/subscription/consumers/'
+        url = upstream['apiUrl'] || 'https://subscription.rhsm.redhat.com/subscription/consumers/'
```

Comment:
What do you think about storing this URL in a constant that way it won't be duplicateD?